cyhy/core/geoloc.py

```diff
@@ -4,13 +4,15 @@
 
 import os
 import sys
+from netaddr import IPSet
```

Comment:
```suggestion
```

Reply:
Ahh yeah this is a more elegant implementation, I think. Good idea. I will need to go into the checks in ip-check and import to re-work how they use it then.